release-notes.md

```diff
@@ -1,3 +1,17 @@
+# Release 1.10.4
+
+## What's New
+* none
+
+## Other changes:
+* none
+
+## Bugs fixed:
+* none
```

Comment:
i'd like to make sure we do a better job of filing/tracking bugs for the release notes :/

Reply:
added the issue information in the release notes